core/src/main/scala/kafka/server/epoch/LeaderEpochFileCache.scala

```diff
@@ -180,7 +180,7 @@ class LeaderEpochFileCache(topicPartition: TopicPartition, leo: () => LogOffsetM
     checkpoint.write(epochs)
   }
 
-  def epochChangeMsg(epoch: Int, offset: Long) = s"New: {epoch:$epoch, offset:$offset}, Latest: {epoch:$latestEpoch, offset$latestOffset} for Partition: $topicPartition"
+  def epochChangeMsg(epoch: Int, offset: Long) = s"New: {epoch:$epoch, offset:$offset}, Old: {epoch:$latestEpoch, offset$latestOffset} for Partition: $topicPartition"
```

Comment:
Nit: it's a little confusing that we log `Old`, but the code is printing the output of `latestEpoch`. Would `Current` perhaps be clearer? I don't feel strongly, so OK to leave it as in the PR if you prefer.